rails_docker/Dockerfile

```diff
@@ -1,32 +1,33 @@
-FROM ruby:2.4.2
+FROM ruby:2.4.2-slim
```

Comment:
@olivierobert i think its better to split Dockerfile based on ENVs, renaming this to `DockerfileCI`.

Reply:
NOt sure about having different Dockerfile as 1) it takes load of time to test and make it work and 2) there are no differences between the `test` (`docker-compose.test.yml`) and `production` (`docker-compose.yml`) environments which are the two envs where the web app is run inside a Docker image. Using the slim version has no downsides in our case as our envs are pretty standard.